src/config/__init__.py

```diff
@@ -54,45 +78,26 @@ def get_value(
     value.set_file(default_config_filename, base_for_paths=True)
 
     # read configuration overrides from the user
-    for user_config_filename in user_config_filenames:
```

Comment:
I had configured it this way to provide the user two options (see below). This is not critical to preserve.
- Use a file in their current directory (e.g., a per-project configuration)
- Use a file in their home directory (e.g., a per-user configuration)

Reply:
i have abolished `user_config_filenames` entirely in favour of a single config file located at ~/.deepirigin/config.yml

i did so because it is very confusing as to where config values are coming from when you have more than one config file, and i found myself wondering why config wasn't changing when i changed it in the file, only to realize there was a second file that was being silently used to ignore the file i was edititing. 

it also makes testing harder, and involves a fair bit of extra code.